src/components/App/App.css

```diff
@@ -0,0 +1,5 @@
+.App {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
```

Comment:
The whole page seems to be a mess with width changes even though the display is flexible. I believe this could be a result of the Navigation component having a fixed position. 

Below is a screenshot with a width of 702px.

![add_motor](https://user-images.githubusercontent.com/43215910/185747368-d1e0e46b-8df4-4d4a-9dde-86f1db7ea49f.png)